Bottom: 23e25f34a44ab6a28ab9a6c7bd55803e9616e253
Top:    8c81211cf6460c594428b52aa1831c82fb9ccf89
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-05-11 16:01:31 +0800

Refresh of vhost-let-vhost_signal-returns

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 2bb6881..4d6ac1a 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1526,15 +1526,12 @@ static bool __vhost_signal(struct vhost_dev *dev, struct vhost_virtqueue *vq)
 }
 
 /* This actually signals the guest, using eventfd. */
-bool vhost_signal(struct vhost_dev *dev, struct vhost_virtqueue *vq)
+void vhost_signal(struct vhost_dev *dev, struct vhost_virtqueue *vq)
 {
 	bool can_coalesce = vq->max_coalesced_buffers && vq->coalesce_usecs;
 	int left;
 	ktime_t now;
 
-	if (!vq->call_ctx)
-		return;
-
 	if (can_coalesce) {
 		now = ktime_get();
 		left = vq->coalesce_usecs -
